.github/workflows/performance-testing.yaml

```diff
@@ -16,6 +16,9 @@ on:
         required: true
         type: string
         default: '20'
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 4 * * *'
```

Comment:
note this schedules the tests to run nightly at 4am GMT

Reply:
Would make sense to include that in comment above?